### ISSUE TITLE
Admin Category/Product Listing icons use color to convey information 

### ISSUE THREAD

**Issue by scottcwilson** [zencart/zencart#6598]
- Meta tags button show customization by turning the white star orange. 
- Attributes button shows presence of attributes by turning black "A" blue. 
- Dollar sign button shows use of PPM and indicates item may be added to the cart. 

Solely using color to convey information is an accessibility error.  
 
It would be good to augment alt text for these icons with language that indicates the state. 
The red/green status icon uses alt text effectively to convey status information, as an example. 

<img width="273" alt="icons" src="https://github.com/user-attachments/assets/b66b9dea-a853-432a-b7b5-9acbb82eeeeb">

**Comment by dbltoe** [comment]
Perhaps change the background to a pattern or x a la https://digital.accessibility.princeton.edu/how/design/color-alone or swap the colors and add a border like the links in the cat tabs menu.

Still, there are many other top level problems on the page that I would work on first.  We have not really delved into the admin, as you are technically not providing the admin to the general public.  Time to work on it just hasn't been there.

And, I am looking at 2.01 vs a 2.1 test.

The little icon for the catalog is an A violation in that it has no text alternative and no name, role, or value.

The search icon and all the status icons are missing name, role, or value.

The product selection form field is missing a label.

None of the test options we mention in the docs will test for the condition you mention but the Siteimprove Accessibility Checker will allow you to apply filters to the page for total-, red/green-, and blue/yellow color blindness.

IMHO the colors are enough of a contrast as to make this an AAA item at most.  A minor note is that it is probably more of a violation that simply color is used to indicate a link when you hover over it.

I would love to "tighten up" the admin but have just not had the time to do so and I hate to just say, "this needs fixin'" without providing the fix.

**Comment by lat9** [comment]
Could we not simply [stack](https://docs.fontawesome.com/v5/web/style/stack/#_top) the FA "ban" (https://fontawesome.com/icons/ban?f=classic&s=solid) icon on top of those to indicate that there are "not" attributes, meta tags or pricing overrides?

**Comment by dbltoe** [comment]
Or, indicate being used with the stack 2x

**Comment by dbltoe** [comment]
I agree with the cogdocile that font size and color contrast might be priorities with mobile phones and tablets being used in such greater numbers.  Legibility prioritized over accessibility might be the way to start.

**Comment by scottcwilson** [comment]
@dbltoe Please fix any issues you find, or log them in separate issues if you don't have time.   Please don't clutter open issues with complaints about other problems; they will get lost when I close this issue. 

This is *my* issue; I will fix it when I have time.  I know it's not a priority, but I want to log it so I don't forget about it.

**Comment by dbltoe** [comment]
perhaps a Post-it note on the monitor if you need a reminder and don't want comments 🥲
